### PR TITLE
add axel, aria2c, fix curl

### PR DESCRIPTION
**Pull request by lizhuoli1126**
axel: add page
aria2c: add page
curl: fix typo

### CODE REVIEW

File: pages/common/aria2c.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`aria2c {{url}}`
+
+- Download from multiple sources
+
+`aria2c {{url_1}} {{url_2}}`
+
+- Download Metalink
+
+`aria2c {{magnet:?xt=urn:btih:...}}`
```
Comment (ostera) [inline pages/common/aria2c.md]: This, and the previous two could be merged into a single one. After all the magnet link is just another link.

```
- Download to a file from one or more supported URLs
`aria2c {{url_1}} {{url_2}}`
```

File: pages/common/axel.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> Download accelerator
+> Supports HTTP, HTTPS, and FTP
+
+- Download a URL to a file
+
+`axel {{url}}`
+
+- Download and rename
+
+`axel {{url}} -o {{filename}}`
```
Comment (ostera) [inline pages/common/axel.md]: Not renaming, but specifying output file.

File: pages/common/axel.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Download with multiple connections
+
+`axel -n {{20}} {{url}}`
+
+- Search for mirrors (can specify numbers)
+
+`axel -S {{5}} {{url}}`
```
Comment (ostera) [inline pages/common/axel.md]: is 5 the amount of mirrors? Then it should be number_of_mirrors

File: pages/common/axel.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`axel {{url}}`
+
+- Download and rename
+
+`axel {{url}} -o {{filename}}`
+
+- Download with multiple connections
+
+`axel -n {{20}} {{url}}`
```
Comment (ostera) [inline pages/common/axel.md]: is 20 the amount of connections? Then it should be something like number_of_connections

File: pages/common/axel.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Search for mirrors (can specify numbers)
+
+`axel -S {{5}} {{url}}`
+
+- Limit download speed (bytes per second)
+
+`axel -s {{10000}} {{url}}`
```
Comment (ostera) [inline pages/common/axel.md]: same here, try to use a name instead of the numeric literal

File: pages/common/curl.md [hunk 6]
```diff
@@ -25,4 +25,4 @@
 
 - pass a user name and password for server authentication
 
-`curl -u myusername:mypassword {{http://localhost}}`
+`curl -u {{username:password}} {{http://localhost}}`
```
Comment (ostera) [inline pages/common/curl.md]: For the sake of consistency, I think this should be

`{{username}}:{{password}}`

We don't have a clearly defined syntax for what happens within the braces, and some people might not understand what the colon is doing there

**Comment by igorshubovych** [comment]
Please next time submit 3 different pull requests.

File: pages/common/axel.md (hunk 2 above)
Comment (lizhuoli1126) [inline pages/common/axel.md]: ok. fix it

**Comment by lizhuoli1126** [comment]
separate to 3 pull requests

**Pull request closed without merging**